lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -169,6 +169,10 @@ module InternalConstants
           'Google::Apis::LoggingV2::LogEntryOperation'
         ]
       }
+
+      # The name of the WriteLogEntriesPartialErrors field in the error details.
+      PARTIAL_ERROR_KEY = 'type.googleapis.com/' \
```

Comment:
Let's break right after the `=`, so the string fits on one line.

Reply:
Changed.